### PR TITLE
[QNN] Align output_scale/zero_point of sigmoid to Torch

### PR DESCRIPTION
**Pull request by zhaoyang-star**
Align output_scale/zero_point of sigmoid to Torch.
cc @masahi

### CODE REVIEW

**Comment by masahi** [comment]
Interesting, this might explain the large accuracy difference I saw between TVM and PT when running quantized mobilenet v3 model.

File: python/tvm/relay/frontend/qnn_torch.py [hunk 1]
```diff
@@ -571,9 +602,12 @@ def quantized_relu(data, input_zero_point):
...
 
-def quantized_sigmoid(data, input_scale, input_zero_point):
-    output_scale = input_scale
-    output_zero_point = input_zero_point
+def quantized_sigmoid(inputs):
+    data = inputs[0]
+    output_scale = _expr.const(inputs[1])
+    output_zero_point = _expr.const(inputs[2])
+    input_scale = _expr.const(inputs[3])
+    input_zero_point = _expr.const(inputs[4])
```
Comment (masahi) [inline python/tvm/relay/frontend/qnn_torch.py:610]: Are you sure that output params occur before input? Your change seem to be calling `_add_output_quant_params_to_sigmoid_op` after we call `_get_quant_param_for_input`, which adds the input qparams first?.

Comment (zhaoyang-star) [reply python/tvm/relay/frontend/qnn_torch.py:610]: Yes. One example printing the `inputs[1-4]` is `0.00390625, 0, 0.09818686544895172, 96
`
The index of output quantized params is set on Line 275: `"aten::sigmoid": (1, 2),`

Comment (zhaoyang-star) [reply python/tvm/relay/frontend/qnn_torch.py:610]: Yes. One example printing the `inputs[1-4]` is `0.00390625, 0, 0.09818686544895172, 96
`
The index of output quantized params is set on Line 275: `"aten::sigmoid": (1, 2),`

**Review by masahi — APPROVED** [review]

**Pull request merged**